### DISCUSSION TITLE
Abuse with backpack

### DISCUSSION THREAD

**Discussion by BurtGumer** [P3pp3rF1y/SophisticatedBackpacks#246] [Q&A] [answered]
Hello! Really love your mod! But I have one problem on my server..
Players learned how to use your backpacks to climb walls..they just placed backpack, jump on it, place the second backpack higher, jump on the second backpack and place the first backpack higher and jump..
It's really dangerous for my RP-server..
We know how to fix it - just add collision for backpack "block", but we can't build it after editing code..
Everything I need - just in BackpackBlock.java - line 65 - edit this function:

public BackpackBlock() {
    super(Properties.of(Material.WOOL).noCollission().noOcclusion().strength(0.8F).sound(SoundType.WOOL));
    registerDefaultState(stateDefinition.any().setValue(FACING, Direction.NORTH).setValue(WATERLOGGED, false).setValue(LEFT_TANK, false).setValue(RIGHT_TANK, false));
  }

Can I ask you just add noCollission() and compile your mod for me?
I will really grateful for this!
My e-mail: mykola.netreba@email.cz
Thanks a lot!

**Comment by P3pp3rF1y** [comment, marked as answer]
Simple answer - "No". That actually breaks the mod. the backpack is supposed to have a collision box and I am pretty sure this would for example break picking up entities from world when they collide with the backpack for pickup upgrade.

Not sure that I understand your use case either but it sounds to me like you may want to ban placement of blocks in an area? (backpack should follow the rules that it can't be placed if the server denies it the placement like with FTB chunks claimed chunks)

Also modifying the mod and distributing it to just a few people if the mod isn't public may be borderline ok, but if you are actually posting the modified mod publicly somewhere or using it for publicly accessible server and haven't posted modified code anywhere you would be breaking the mod's GPL license. Not that I would take it that seriously in this case but some other modder could so just a word of advice.

**Reply by BurtGumer** [reply]
Thanks for answer. 
We don't use any upgrades, so I don't thing that would break something..

And I don't really want to share mod. I just use it on my small server without any donate or something like that. Just DayZ server in minecraft, nothing else.
Unfortunately, we can't simply deny placement because I'd like to allow place backpacks on bases, for example, but we have really huge map to deny placement on regions..

**Comment by HeavyDeception** [comment]
@BurtGumer , @P3pp3rF1y Could you give the Backpack a rule so that it has to have a block under it or it will break kind of like a pumpkin has to have a block under it to be placed.
